package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-schema-designer",
-  "version": "0.0.18",
+  "version": "0.0.19",
```

Comment:
I don't know if I can check this but did you add a tag with the version #

Reply:
Yeah it's been tagged